strawberryfields/apps/qchem/dynamics.py

```diff
@@ -92,51 +92,40 @@
 from strawberryfields.utils import operation
 
 
-def evolution(modes: int):
-    r"""Generates a custom ``sf`` operation for performing the transformation
-    :math:`U(t) = U_l e^{-i\hat{H}t/\hbar} U_l^\dagger` on a given state.
+def TimeEvolution(t: float, w: np.ndarray):
+    r"""An ``sf`` operation for performing the transformation
```

Comment:
Same comment as in #451 : I'm not a big fan of using "An `sf` operation". I prefer just "an operation"